src/components/JoinUs/elements.js

```diff
@@ -30,7 +30,7 @@ export const Section = ({ children, greyBg, ...props }) => {
     return content
   }
 
-  return <GreyBG static>{content}</GreyBG>
+  return <GreyBackground static>{content}</GreyBackground>
```

Comment:
no need for the `static` prop anymore, right?

Reply:
Good point! Removed :D